1-bootstrap/versions.tf

```
@@ -22,6 +22,11 @@ terraform {
       version = ">= 6.6, < 7"
     }
 
+    google-beta = {
+      source  = "hashicorp/google-beta"
+      version = "6.20.0"
```

Probably should be more like:

```suggestion
      version = ">=  6.20.0, < 7"
```